src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -22,6 +22,7 @@
     public static final Prefix PREFIX_NEW_TAG = new Prefix("nt/");
 
     public static final Prefix PREFIX_TITLE = new Prefix("ttl/");
+    public static final Prefix PREFIX_NEW_TITLE = new Prefix("nttl/");
```

Comment:
Can we recycle the prefixes used in the addlog command for editlog as well? Since there is no need to differentiate between new title and old title for edit logs. Similarly for descriptions as well.

Reply:
Can, will do.